python/pyspark/sql/tests/test_udtf.py

```diff
@@ -2812,6 +2814,15 @@ def eval(self):
                 with self.assertRaisesRegex(PythonException, "UDTF_ARROW_TYPE_CAST_ERROR"):
                     udtf(TestUDTF, returnType=ret_type)().collect()
 
+    def test_udt_as_udtf_return_type(self):
+        class TestUDTF:
+            def eval(self):
+                yield ExamplePoint(0, 1),
+
+        schema = StructType().add("point", ExamplePointUDT())
+        [row] =udtf(TestUDTF, returnType=schema, useArrow=True)(lit()).collect()
```

Comment:
```suggestion
        [row] = udtf(TestUDTF, returnType=schema, useArrow=True)(lit()).collect()
```
Can we directly use `udtf(...).collect()` here?

Reply:
DOne